persistence-jdbc/scaladsl/src/multi-jvm/scala/com/lightbend/lagom/javadsl/persistence/jdbc/JdbcClusteredPersistentEntitySpec.scala

```diff
@@ -48,12 +52,17 @@ class JdbcClusteredPersistentEntitySpec extends AbstractClusteredPersistentEntit
     Option(h2).foreach(_.stop())
   }
 
-  def testEntityReadSide = injector.instanceOf[JdbcTestEntityReadSide]
+  override def environment: Environment = JdbcClusteredPersistentEntityConfig.environment
+
+  override lazy val applicationLifecycle: ApplicationLifecycle = new DefaultApplicationLifecycle
```

Comment:
`DefaultApplicationLifecycle.stop()` will need to be executed in the after tests are finished, this will shutdown connection pools, and whatever else has registered shutdown callbacks.